app/scripts/background.js

```diff
@@ -801,6 +801,9 @@ export function setupController(
           case ApprovalType.SnapDialogConfirmation:
             controller.approvalController.accept(id, false);
             break;
+          case 'snap_manageAccounts:confirmation':
```

Comment:
We try and maintain all approval types used across both clients in the `ApprovalType` enum defined in `@metamask/controller-utils` in the core repository.

I don't want to block this work but it would be great if we could create the necessary core PR and then make a subsequent PR to use the new enum value here.

Reply:
Made an issue to track this work: https://github.com/MetaMask/accounts-planning/issues/70